src/pages/console/auth/import.mdx

```diff
@@ -35,26 +45,34 @@ These roles are usually automatically configured when you create a new identity
 
 [Learn more about getting started with Identity Pools.](https://docs.aws.amazon.com/cognito/latest/developerguide/getting-started-with-identity-pools.html)
 
-## Import and update your source code
+We can also use Amplify CLI to [import an Cognito identity pool resource](/cli/auth/import/).
```

Comment:
In order to import the identity pool, do you also need to import the user pool? Might be worth pointing that out if so.

Reply:
Looks like a typo "an Cognito identity pool resource" should be "a Cognito identity pool resource"